### PR TITLE
Add tests for the AddUserForm component

### PR DESCRIPTION
**Pull request by bdj61383**

### CODE REVIEW

File: src/components/AddUserForm/AddUserForm.test.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import AddUserForm from './AddUserForm';
+
+/* eslint-env mocha */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable prefer-const */
```
Comment (JeffreyATW) [inline src/components/AddUserForm/AddUserForm.test.js]: Move the first two comments to the top of the file. The third is unnecessary if you do `const props` on line 11 instead.

File: src/components/AddUserForm/AddUserForm.test.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+describe('Layout', () => {
```
Comment (JeffreyATW) [inline src/components/AddUserForm/AddUserForm.test.js]: Should be `'AddUserForm'`.

File: src/components/AddUserForm/AddUserForm.test.js (hunk 1 above)
Comment (bdj61383) [reply src/components/AddUserForm/AddUserForm.test.js]: I wanted to ask you about that.  What's the accepted use of the `const` keyword in Javascript?  Here we're constructing an object with a single property, but then for future tests we're mutating that object.  It didn't seem like a constant to me, even though ES-lint didn't complain when I mutated the object.

Comment (JeffreyATW) [reply src/components/AddUserForm/AddUserForm.test.js]: You can mutate constants. You just can't reassign them.

I think the proper fix in this case is to reassign `let props` before every test (within a `beforeEach` block).

Comment (bdj61383) [reply src/components/AddUserForm/AddUserForm.test.js]: Yeah, that makes sense.

Comment (bdj61383) [reply src/components/AddUserForm/AddUserForm.test.js]: I meant the beforeEach block makes sense.  Having constants that can be mutated still doesn't make much sense to me, but I'll adjust 😛.  So `const foo = {}; foo.bar = 'bar';` is fine, but `const foo = {}; foo = {bar: 'bar'};` is not, correct?

Comment (JeffreyATW) [reply src/components/AddUserForm/AddUserForm.test.js]: Correct.

**Pull request merged**